content/zh-cn/docs/tasks/extend-kubernetes/socks5-proxy-access-api.md

```diff
@@ -64,7 +64,7 @@ Figure 1 represents what you're going to achieve in this task.
 graph LR;
 
   subgraph local[Local client machine]
-  client([client])-- local <br> traffic .->  local_ssh[Local SSH <br> SOCKS5 proxy];
+  client([client])-. local <br> traffic .->  local_ssh[Local SSH <br> SOCKS5 proxy];
```

Comment:
The mermaid diagram still isn't displaying correctly in the [preview (here)](https://deploy-preview-45157--kubernetes-io-main-staging.netlify.app/zh-cn/docs/tasks/extend-kubernetes/socks5-proxy-access-api/). It seems that applying the fix on [Line 89](https://github.com/kubernetes/website/blob/b50df526e05dc2e84ee15829a9b9dbe994d77787/content/zh-cn/docs/tasks/extend-kubernetes/socks5-proxy-access-api.md?plain=1#L89) might resolve the issue.

Reply:
Done and some fix